server/middleware/setUpStaticResources.ts

```diff
@@ -19,6 +19,7 @@ export default function setUpStaticResources(): Router {
     '/assets',
     '/assets/stylesheets',
     '/assets/js',
+    '/server/assets',
```

Comment:
Just checking - should these be copied over?

Reply:
Yes this should be removed. I need to work on migrating the application js file to ts.